### DISCUSSION TITLE
Overriding Views not work in my Admin theme, only work in Startup project

### DISCUSSION THREAD

**Discussion by scil** [OrchardCMS/OrchardCore#10729] [Q&A]
I'd like to override the image manage page ( OrchardCore.Modules\OrchardCore.Media\Views\Admin\Index.cshtml).

I have enable feature AdminTemplates.

it not work in my admin theme, the dir structure is (I also tried  dir `Shared`  instead of `Admin`)
```
Scil.Theme.TheAdmin
  Views
    OrchardCore.Media
      Admin
        Index.cshtml
```

But it works in my startup project:
```
Scil.Application.Web
  Views
    OrchardCore.Media
      Admin
        Index.cshtml
```

**Comment by ns8482e** [comment]
You can not override MVC Views using `AdminTemplates` just shape templates.

`OrchardCore.Modules\OrchardCore.Media\Views\Admin\Index.cshtml` is MVC view 

You can only override MVC view in theme or in startup project  ie.`Views\OrchardCore.Media\Admin\Index.cshtml`

**Comment by scil** [comment]
I did overried the file in my theme or startup project.

In startup project, it worked!  

But in theme project, it not.
